components/UserObserver.php

```diff
@@ -89,10 +90,11 @@ public function updateUser($params) :User
 					\Yii::log("User " . $username . " not found in the CSD database.", \CLogger::LEVEL_ERROR);
```

Comment:
from line 78 to 83, is it possible to update the contact id with the first `saveUser` at line 66 by changing the code order?

Reply:
I've followed the code from V1.19. The contact id does not have to be changed every time.

Reply:
Yeah but in this part of the code you are updating the contact with the info from the user, reorganising the code to be simpler would be better for readability and maintainability.